aeson-deriving.cabal

```diff
@@ -7,7 +7,7 @@ cabal-version: 1.12
 -- hash: 05a421de32621a1d52633e1311198dbc32abe2ea1f27c1e0f003d14a579b97e5
```

Comment:
The `.cabal` file is generated from `package.yaml`, but you seem to have edited the `.cabal` file directly. You should make the changes to `package.yaml` and then re-generate the `.cabal` file from it with `hpack`.